docs/Secure-Coding-Guide-for-Python/CWE-707/CWE-175/noncompliant02.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-FileCopyrightText: OpenSSF project contributors
+# SPDX-License-Identifier: MIT
+""" Non-compliant Code Example """
+import io
+ 
+LOREM = """Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```

Comment:
Got a lot of "Line too long" from Pylint... not sure if we care about this though:
Line too long (456/100)Pylint[C0301:line-too-long](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/line-too-long.html)
(constant) LOREM: Literal['Lorem ipsum dolor sit amet, consectetur adipiscing…']

Got that for lines 6,7,8 and 18